sites/example-project/src/components/ui/Alert.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+  // Based on the alert from FlowBite: https://flowbite.com/docs/components/alerts/
+  /**
+   * Defines the color of the alert
+   * @type {"default" | "info" | "danger" | "success" | "warning"}
+   */
+  export let status = "default";
+</script>
+
+<div class="alert {status}" role="alert">
+  <slot />
+</div>
+
+<style lang="postcss">
+  .alert {
+    @apply p-4 mb-4 text-sm rounded text-gray-800 bg-gray-50 font-medium;
+    &.info {
+      @apply text-blue-800 bg-blue-50;
+    }
+    &.danger {
+      @apply text-red-800 bg-red-50;
+    }
+    &.success {
+      @apply text-green-800 bg-green-50;
+    }
+    &.warning {
+      @apply text-yellow-800 bg-yellow-50;
+    }
+  }
```

Comment:
I _really_ like this approach to applying the TW styles. Much more svelte-y

Reply:
There are pro's and cons to this; typically I use this whenever there are dynamic styles (e.g. here we have the `status`), because it works well against tailwind's auto-pruning of styles.

One con is that this can really slow down the build time if you use it everywhere, as (if I recall correctly), it will process each of style tag separately. [There are some docs on the topic](https://tailwindcss.com/docs/reusing-styles#avoiding-premature-abstraction)